nomenclature/definitions/region/subcountries.yaml

```diff
@@ -302,28 +302,58 @@ United Kingdom|North Sea:
     ehighway2050: 106_NS+107_NS+108_NS+109_NS
     plan4res: NSUK
     
-# List of electricity price regions in Norway
-Norway|Oslo:
-   definition: this is the electricity price region 1 in Norway 
+# List of subregions in Norway, designed to best decribe the hydropower system in Norway
+Norway|OSTLAND:
+   definition: this is region N1 in the HydroCen dataset, these regions are based on the norwegian electricicty regions but with finer granularity (and some changes) 
    country: Norway
    source: https://energifaktanorge.no/en/norsk-energiforsyning/kraftmarkedet/
 
-Norway|Kristiansand:
-   definition: this is the electricity price region 2 in Norway 
+Norway|SOROST:
+   definition: this is region N2 in the HydroCen dataset, these regions are based on the norwegian electricicty regions but with finer granularity (and some changes) 
    country: Norway
    source: https://energifaktanorge.no/en/norsk-energiforsyning/kraftmarkedet/
 
-Norway|Trondheim:
-   definition: this is the electricity price region 3 in Norway 
+Norway|HALLNGDAL:
```

Comment:
Yes... Just doublechecked HALLINGDAL

Reply:
Then there is an "I" missing in the region name... Also, are you sure that you want to have all regions written in all-caps?

Reply:
Yes, the naming in all the HydroCen datasets are all capital... I thought then I might best stick to it?

Reply:
For consistency with other region names and for better readability, I would use "normal" spelling rather than all-caps. If one wants to write a script to automatically convert from HydroCen to openENRTANCE in a workflow, a user can easily generate a mapping like `{'HALLINGDAL': 'Norway|Hallingdal'}`.

Reply:
I have changed that now (if I succeeded in providing the right file.. )